react-native-community/cli
# fix: Add 'code-insiders' to launchEditor

Summary:
---------

Visual Studio Code Insiders has a seperate editor name than Visual Studio Code, but has the same signature, so this PR adds 'code-insiders' to getArgumentsForLineNumber


Test Plan:
----------

1. Install https://code.visualstudio.com/insiders/
2. Run `> Shell Command: Install 'code-insiders' command in PATH` in command pallet
3. Change `$REACT_EDITOR` to 'code-insiders'
4. Throw an error in development of a react-native cli app. 
5. Click on the logbox error.
6. See that error is opened

Checklist
----------
